Switch calls of fgets from using magic numbers to using sizeof() to
ensure that the size passed to fgets is consistent with the size
allocated by the compiler.

diff -ru a/plugins/ADM_videoFilters/Srt/ADM_vidSRTload.cpp b/plugins/ADM_videoFilters/Srt/ADM_vidSRTload.cpp
--- a/plugins/ADM_videoFilters/Srt/ADM_vidSRTload.cpp	2009-12-19 20:41:13.000000000 +0000
+++ b/plugins/ADM_videoFilters/Srt/ADM_vidSRTload.cpp	2010-10-02 21:02:59.000000000 +0000
@@ -86,7 +86,7 @@
 	return 0;
 
   }
-  while (fgets (string, 200, _fd))
+  while (fgets (string, sizeof(string), _fd))
     {
       _line++;
     }
@@ -104,7 +104,7 @@
 //
   for (uint32_t i = 0; i < _line; i++)
     {
-      fgets (string, ADM_RAW, _fd);
+      fgets (string, sizeof(string), _fd);
       //printf("\n in : %s ",string);
       if (string[0] != '{')
 	continue;
@@ -250,7 +250,7 @@
   // first cound how many line
   line = 0;
   _line = 0;
-  while (fgets (string, 300, _fd))
+  while (fgets (string, sizeof(string), _fd))
     line++;
   printf ("\n subs : %ld lines\n", line);
   // rewind
@@ -272,7 +272,7 @@
   for (uint32_t i = 0; i < line; i++)
     {
 	current=&_subs[_line];
-	fgets (string, ADM_RAW, _fd);
+	fgets (string, sizeof(string), _fd);
 	ADM_utfConv(final,string,strlen(string),&finallen);
 	// Purge cr/lf
 	switch (state)
